tests/integration/tests/test_cloudbeat_integration.py

```diff
@@ -90,6 +90,7 @@ def test_elastic_index_exists(kspm_client, match_type):
 
 
 @pytest.mark.pre_merge
+@pytest.mark.skip(reason="https://github.com/elastic/cloudbeat/issues/2383")
 @pytest.mark.order(4)
 @pytest.mark.dependency(depends=["test_cloudbeat_pod_exist"])
 def test_leader_election(fixture_data, kspm_client, cloudbeat_agent, k8s):
```

Comment:
The `pre_merge-kind-multi` CI will fail because it executes `pull_request_target`.